tasks/setup.yaml

```
@@ -57,36 +57,100 @@ tasks:
     description: Print the default keycloak 'admin' password to standard out (if INSECURE_ADMIN_PASSWORD_GENERATION was used on uds-core)
     actions:
       - description: Print the default keycloak admin password to standard out (if available)
-        cmd: ./uds zarf tools kubectl get secret -n keycloak keycloak-admin-password -o jsonpath='{.data.password}' | base64 -d
+        cmd: |
+          USERNAME=$(uds zarf tools kubectl get secret keycloak-admin-password -n keycloak -o jsonpath='{.data.username}' | base64 --decode)
+          PASSWORD=$(uds zarf tools kubectl get secret keycloak-admin-password -n keycloak -o jsonpath='{.data.password}' | base64 --decode)
 
-  - name: create-doug-user
-    description: Creates a user named 'doug' in the uds realm of keycloak (using the default admin account)
+          printf "\033[31m!!! Please ensure you're not running this in CI !!!\033[0m\n"
+          printf "Keycloak Admin Username: \033[31m %s \033[0m\n" "$USERNAME"
+          printf "Keycloak Admin Password: \033[31m %s \033[0m\n" "$PASSWORD"
+
+  - name: keycloak-admin-user
+    description: Sets up the Keycloak admin user for dev/testing if not already created
+    actions:
+      - description: Create Keycloak Admin User
+        cmd: |
+          # Check if the secret exists
+          if ./zarf tools kubectl get secret keycloak-admin-password -n keycloak > /dev/null 2>&1; then
```

```suggestion
          if ./uds zarf tools kubectl get secret keycloak-admin-password -n keycloak > /dev/null 2>&1; then
```

Inside a task it has to be `./uds zarf` you can only use `./zarf` in a zarf.yaml

---

`./zarf` actually should still work in tasks (it does for our uds-core version of this, although `./uds zarf` is probably better for consistency sake.

---

Okay that's good to know i think i tracked down the previous conversation that was making me think it wouldn't. Most of the confusion around this in the past was actually related to folks trying to use `./uds zarf` within a zarf.yaml.

In any case, for consistency i would still ask they get changed to `./uds zarf`